web/src/classic/components/molecules/Dashboard/Workspace.tsx

```diff
@@ -119,4 +124,11 @@ const StyledAvatar = styled(Avatar)`
   margin-right: ${metricsSizes["s"]}px;
 `;
 
+const CustomText = styled(Text)`
+  max-width: 150px;
```

Comment:
How do we get the `150px` here? It's a card and the content should be flex enough i think. Please double confirm with Red about the design.

Reply:
Thanks Shaun. I initially used an arbitrary value so that the text does not take the full-width if the workspace name is too long. Red asked we use `200px` here and I have updated.